csrankings-p.csv

```diff
@@ -996,7 +996,6 @@ Prashant V. Krishnamurthy,University of Pittsburgh,http://www.sci.pitt.edu/facul
 Prashanth L. A.,IIT Madras,http://www.cse.iitm.ac.in/~prashla,Q1YXWpoAAAAJ
 Prashanth Lakshmanrao Ananthapadmanabharao,IIT Madras,http://www.cse.iitm.ac.in/~prashla,Q1YXWpoAAAAJ
 Prasun Dewan,University of North Carolina,http://www.cs.unc.edu/~dewan,rRxC18AAAAAJ
-Prasun Sinha,Ohio State University,http://web.cse.ohio-state.edu/~prasun,jOX_9bQAAAAJ

```

Comment:
Moved to industry? If so, please add entry to `old/industry.csv` with affiliation.